snap/local/bin/run.sh

```diff
@@ -8,9 +8,7 @@ docker create \
   --name "$container_name" \
   --log-driver none \
   --env-file "$SNAP_COMMON/conf.env" \
-  --device=/dev/spidev0.0:/dev/spidev0.0 \
-  --device=/dev/gpiochip0:/dev/gpiochip0 \
-  --device=/dev/gpiochip4:/dev/gpiochip4 \
+  --privileged \
```

Comment:
```suggestion
  --device=/dev/$SPI:/dev/$SPI \
  --device=/dev/$GPIO:/dev/$GPIO \
```

Repurpose the existing envs for this.

Set defaults from an install hook. E.g. https://github.com/canonical/openthread-border-router-snap/blob/main/snap/hooks/install